client/my-app/src/components/ExpenseModal/ExpenseModal.tsx

```diff
@@ -0,0 +1,66 @@
+import { useState, Dispatch, SetStateAction, FC } from "react";
+import { patchExpense } from "../../api/expenses";
+import { detectType } from '../NewExpenseForm'
+
+type Props = {
+  expense: Expense,
+  onClose: Dispatch<SetStateAction<boolean>>,
+};
+
+export const ExpenseModal: FC<Props> = ({ expense, onClose }) => {
+  const [dataForUpdate, setDataForUpdate] = useState({
+    title: expense.title,
+    amount: expense.amount,
+    category: expense.category,
+    spentAt: expense.spentAt,
+    note: expense.note || '',
+  });
+
+  const handleSubmit = async() => {
+    try {
+      await patchExpense(expense.id, dataForUpdate);
+    } catch (err: any) {
+      throw new Error(err);
+    }
+
+    onClose(false);
```

Comment:
`onClose` is not descriptive name at all
and follow name conventions for boolean